src/App/Fossa/Analyze.hs

```diff
@@ -340,24 +342,33 @@ analyze cfg = Diag.context "fossa-analyze" $ do
         if (fromFlag BinaryDiscovery $ Config.binaryDiscoveryEnabled $ Config.vsiOptions cfg)
           then analyzeDiscoverBinaries basedir filters
           else pure Nothing
+  let ficusStrategies =
+        [ strategy
+        | (strategy, enabled) <-
+            [ (FicusStrategySnippetScan, enableSnippetScan)
+            , (FicusStrategyVendetta, enableVendetta)
+            ]
+        , enabled
+        ]
   maybeFicusResults <-
     Diag.errorBoundaryIO . diagToDebug $
-      if not enableSnippetScan
+      if null ficusStrategies
         then do
-          logInfo "Skipping ficus snippet scanning (--x-snippet-scan not set)"
+          logInfo "Skipping ficus scanning (--x-snippet-scan and/or x-vendetta not set)"
```

Comment:
This log line has always bugged me. It's just noise, IMO, and we shouldn't be calling out that we a customer is not using a feature. Should we just remove it altogether?